roofit/roofitcore/inc/RooDataHist.h

```diff
@@ -231,6 +231,9 @@ class RooDataHist : public RooAbsData, public RooDirItem {
   void importTH1Set(const RooArgList& vars, RooCategory& indexCat, std::map<std::string,TH1*> hmap, double initWgt, bool doDensityCorrection) ;
   void importDHistSet(const RooArgList& vars, RooCategory& indexCat, std::map<std::string,RooDataHist*> dmap, double initWgt) ;
 
+  // get_wgt is necessary because it is overridden in RooFitExtensions' RooExpandedDataHist
+  virtual double get_wgt(std::size_t idx) const { return _wgt[idx]; }
```

Comment:
```suggestion
  // get_wgt is necessary because it is overridden in RooFitExtensions' RooExpandedDataHist
  double get_wgt(std::size_t idx) const { return _wgt[idx]; }
```
This is a potentially dangerous behavior change: even before 23d861273, the `get_wgt` function was not virtual.
That means some `RooDataHist::get_wgt` calls in RooFitExt that were not redirecting to the child class implementation now do exactly that.

Is that the desired behavior?